shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/EventReceiverConfiguration.java

```diff
@@ -24,14 +24,14 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
-import com.hotels.shunting.yard.receiver.kinesis.messaging.KinesisMessageReaderFactory;
+import com.hotels.shunting.yard.receiver.sqs.messaging.SqsMessageReaderFactory;
 
 @Configuration
 @ConfigurationProperties(prefix = "event-receiver")
 public class EventReceiverConfiguration {
 
   private Map<String, String> configurationProperties;
-  private @NotNull String messageReaderFactoryClass = KinesisMessageReaderFactory.class.getName();
+  private @NotNull String messageReaderFactoryClass = SqsMessageReaderFactory.class.getName();
```

Comment:
So this set the default messageReader to SQS. which we can override in config correct?

Reply:
My understanding is that this is hardcoded not (currently?) configurable. If so, it's not great but it's OK for now.

Reply:
That is true but the configuration was not making it read SqsMessageReaderFactory. It was basically hardcoded in here. I think we are not going to be using Kinesis or Kafka, so at the moment its ok for it to be hardcoded.

Reply:
yeah I don't mind we change the default